test/apiv2/20-containers.at

```diff
@@ -98,9 +98,41 @@ t GET libpod/images/newrepo:v2/json 200 \
   .Comment=bar \
   .Config.Cmd[-1]="/bin/foo"
 
+# Create a container for testing the container initializing later
+podman create -t -i --name myctr alpine ls
+
+# Check configuration before initializing
+t GET libpod/containers/myctr/json 200 \
+  .Id~[0-9a-f]\\{64\\} \
+  .State.Status="configured" \
+  .State.Pid=0 \
+  .ResolvConfPath="" \
+  .HostnamePath="" \
+  .HostsPath="" \
+  .NetworkSettings.SandboxKey=""
+
+cpid_file=$(jq -r '.ConmonPidFile' <<<"$output")
+userdata_path=$(dirname $cpid_file)
+
+# Initializing the container
+t POST libpod/containers/myctr/init '' 204
+
+# Check configuration after initializing
+t GET libpod/containers/myctr/json 200 \
+  .Id~[0-9a-f]\\{64\\} \
+  .State.Status="created" \
+  .State.Pid~[0-9]\\{5\\} \
```

Comment:
Careful - this is going to pass only if you're lucky. I would prefer to see `{1,8}`